finagle-http2/src/test/scala/com/twitter/finagle/http2/transport/client/UpgradeRequestHandlerTest.scala

```diff
@@ -56,7 +56,7 @@ class UpgradeRequestHandlerTest extends FunSuite {
       val request = new DefaultFullHttpRequest(HttpVersion.HTTP_1_1, HttpMethod.GET, "/")
       channel.writeOutbound(request)
 
-      assert(stats.counters.get(Seq("upgrade", "attempt")) == Some(1l))
+      assert(stats.counters.get(Seq("upgrade", "attempt")).contains(1L))
```

Comment:
what was wrong with the original version?

Reply:
Not sure. Happy to revert.

Reply:
`1l` is depricated due to poor legibility, so `1l` -> `1L`, otherwise reverted. could be `Some(1)` too, `Some(1L) == Some(1)`, regardless of what you may think of that. The bikest of sheds.